### DISCUSSION TITLE
Trying out GitHub's discussions

### DISCUSSION THREAD

**Discussion by fstamour** [fstamour/cl-car#5] [General]
![Screenshot_20231127-225629_Firefox](https://github.com/fstamour/trivial-cdr/assets/2881922/2846feb6-52e6-41a4-9228-ab4fd6663019)

**Comment by fstamour** [comment]
> ![Screenshot_20231127-225629_Firefox](https://private-user-images.githubusercontent.com/2881922/286117367-2846feb6-52e6-41a4-9228-ab4fd6663019.jpg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTEiLCJleHAiOjE3MDExNDQxNjksIm5iZiI6MTcwMTE0Mzg2OSwicGF0aCI6Ii8yODgxOTIyLzI4NjExNzM2Ny0yODQ2ZmViNi01MmU2LTQxYTQtOTIyOC1hYjRmZDY2NjMwMTkuanBnP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQUlXTkpZQVg0Q1NWRUg1M0ElMkYyMDIzMTEyOCUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyMzExMjhUMDM1NzQ5WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9ZTQxMWNkNjg5ZTcxZmZlMjg0NjA5ZWE1ZGE0MjE0ZWI2MGY1YjE5YTIzMmU3MDM3NDExMTU5MDdiNjViZWZiYyZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.JwZRLwCKTDgxQgwGwWn5KgmQRwhJlm4EteSH7TFZZpw)

Here I was showing the types of discussions

Now I'm showing how "quoting" looks like

**Reply by fstamour** [reply]
That's weird (but somewhat logical): you cannot "reply" to the original post, ony to comments.

I'm just glad there are threads to be honest

**Reply by Gleefre** [reply]
Yep, threads are cool :)
Although IMHO discussions are *bit too* compilcated [ at least by default ] :/

**Reply by Gleefre** [reply]
But you can only have one level of nesting, it is more like discord than reddit

**Reply by Gleefre** [reply]
Sadly you also can't collapse a thread :/

**Reply by Gleefre** [reply]
Hm, but it works better on mobile. Meh.

**Comment by fstamour** [comment]
Hey look, you can pin stuff:

![Screenshot_20231127-230145_Firefox](https://github.com/fstamour/trivial-cdr/assets/2881922/f358366a-f878-48d7-a634-0a96a7b59a78)

**Reply by Gleefre** [reply]
Cool! Can you pin a specific comment/thread in a discussion?

**Reply by fstamour** [reply]
Don't think so

![Screenshot_20231127-233150_Firefox](https://github.com/fstamour/trivial-cdr/assets/2881922/6522b5e3-5d8d-4798-a21c-5c1ce299fa95)
